lib/public/views/QcFlags/ActiveColumns/qcFlagsActiveColumns.js

```diff
@@ -14,6 +14,7 @@
 import { h } from '/js/src/index.js';
 import { formatTimestamp } from '../../../utilities/formatting/formatTimestamp.js';
 import { qcFlagTypeColoredBadge } from '../common/qcFlagTypeColoredBadge.js';
+import { formatQcFlagCreatedBy } from '../common/formatQcFlagCreatedBy.js';
```

Comment:
This should probably not go into common directory. Same for qcFlagTypeColoredBadge, I have not noticed before